sklearn/ensemble/forest.py

```diff
@@ -1008,7 +1008,7 @@ class RandomForestRegressor(ForestRegressor):
 
     oob_score : bool, optional (default=False)
         whether to use out-of-bag samples to estimate
-        the generalization error.
+        the generalization R-squared.
```

Comment:
maybe the "``r2_score`` on unseen data"? Not sure. In the docs of ``r2_score`` we write ``R^2`` http://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html